src/main/blackbox/blackbox.c

```diff
@@ -1402,8 +1403,12 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_JITTER_FACTOR, "%d",  currentPidProfile->feedforward_jitter_factor);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_BOOST, "%d",          currentPidProfile->feedforward_boost);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_FEEDFORWARD_MAX_RATE_LIMIT, "%d", currentPidProfile->feedforward_max_rate_limit);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_ANGLE_FEEDFORWARD, "%d",          currentPidProfile->pid[PID_LEVEL].F);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_LEVEL_GYRO_FF_SCALE, "%d",        currentPidProfile->levelGyroFeedforwardScale);
 #endif
-
+        BLACKBOX_PRINT_HEADER_LINE("angle_dterm_lpf_hz", "%d",              currentPidProfile->angle_dterm_lpf_hz);     
+        BLACKBOX_PRINT_HEADER_LINE("roll_level_expo", "%d",                 currentControlRateProfile->levelExpo[FD_ROLL]);
+        BLACKBOX_PRINT_HEADER_LINE("pitch_level_expo", "%d",                currentControlRateProfile->levelExpo[FD_PITCH]);
```

Comment:
Please rename these two expo values so that they start with 'level', and use the param_name method.

Reply:
Happy to rename the variables once functionality is agreed. I'm uncomfortable renaming long standing exisiting variables as part of a feature PR. Better to do that separately I think. I've seen PRs in other projects derailed before by conversations on what variables should be called!